README.md

```
@@ -1,130 +1,164 @@
 
-Run Llama cpp python server CPU
-```bash
-docker run -v LLM_SERVER:/models smartappli/llama-cpp-python-server-cpu
-```
+## 4) Construire les images Docker
 
-or
+> Les commandes suivantes sont à exécuter depuis le dossier `Docker/`.
 
-Run Llama cpp python server GPU
+### Image CPU
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)